File: Sample/userCode.py
xPoint = float(input())
yPoint = float(input())


if (xPoint == 0) and (yPoint == 0):
	print(0)
elif xPoint == 0:
	print(5)
elif yPoint == 0:
	print(6)
elif (xPoint > 0) and (yPoint > 0):
	print(1)
elif (xPoint < 0) and (yPoint > 0):
	print(2)
elif (xPoint < 0) and (yPoint < 0):
	print(3)
elif (xPoint > 0) and (yPoint < 0):
	print(4)

File: Sample/userCode.php
<?php

function solve() {
    // Read input data
    $X = (float) fgets(STDIN);
    $Y = (float) fgets(STDIN);

    // Solve the problem
    if ($X == 0 && $Y == 0) {
        return 0;
    } else if ($X == 0) {
        return 5;
    } else if ($Y == 0) {
        return 6;
    } else if ($X > 0 && $Y > 0) {
        return 1;
    } else if ($X < 0 && $Y > 0) {
        return 2;
    } else if ($X < 0 && $Y < 0) {
        return 3;
    } else {
        return 4;
    }
}

echo solve();
?>